src/TLuaInterpreter.cpp

```diff
@@ -6801,30 +6844,19 @@ int TLuaInterpreter::exists( lua_State * L )
     QString name = _name.c_str();
     if( type == "timer")
     {
-        QMap<QString, TTimer *>::const_iterator it1 = pHost->getTimerUnit()->mLookupTable.find( name );
-        while( it1 != pHost->getTimerUnit()->mLookupTable.end() && it1.key() == name )
-        {
-            cnt++;
-            it1++;
-        }
+        cnt += pHost->getTimerUnit()->mLookupTable.count( name );
     }
     else if( type == "trigger")
     {
-        QMap<QString, TTrigger *>::const_iterator it1 = pHost->getTriggerUnit()->mLookupTable.find( name );
-        while( it1 != pHost->getTriggerUnit()->mLookupTable.end() && it1.key() == name )
-        {
-            cnt++;
-            it1++;
-        }
+        cnt += pHost->getTriggerUnit()->mLookupTable.count( name );
     }
     else if( type == "alias")
     {
-        QMap<QString, TAlias *>::const_iterator it1 = pHost->getAliasUnit()->mLookupTable.find( name );
-        while( it1 != pHost->getAliasUnit()->mLookupTable.end() && it1.key() == name )
-        {
-            cnt++;
-            it1++;
-        }
+        cnt += pHost->getAliasUnit()->mLookupTable.count( name );
+    }
+    else if( type == "keybind")
+    {
+        cnt += pHost->getKeyUnit()->mLookupTable.count( name );
```

Comment:
Related to this, is there an exists() method that could be called instead of count? We don't need the count for this function, and I suspect exists would be faster because it could short circuit once it finds one. They both technically happen in O(n) time, but a count method will never finish early. 

It would make more semantic sense if exists returned a bool (or so nothing breaks, a value of 0 or 1).